client/fe/webpack/moduleRules.js

```diff
@@ -80,7 +80,7 @@ if (config.cssPurge.enabled) {
       content: config.cssPurge.paths,
       defaultExtractor (content) {
         const contentWithoutStyleBlocks = content.replace(/<style[^]+?<\/style>/gi, '')
-        return contentWithoutStyleBlocks.match(/[A-Za-z0-9-_/:]*[A-Za-z0-9-_/]+/g) || []
+        return contentWithoutStyleBlocks.match(/[A-Za-z0-9-_/:]*[A-Za-z0-9-_/.]+/g) || []
```

Comment:
shouldn't the `.` be escaped?

Reply:
Not when it's in a character group.

Reply:
within square brackets, the dot loses its special meaning and is treated as a literal dot.